recipe/build.sh

```diff
@@ -23,6 +23,18 @@ elif [[ "$target_platform" == osx-64 ]]; then
 		-DENABLE_PROGRAMS=OFF \
 		-DENABLE_TESTING=OFF \
   		-D_POSIX_C_SOURCE=199309L
+elif [[ "$target_platform" == linux-aarch64 ]]; then
+ 	cmake -B build -S . \
+		${CMAKE_ARGS} \
+		-DCMAKE_INSTALL_PREFIX=$PREFIX \
+		-DCMAKE_INSTALL_LIBDIR=lib \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_VERBOSE_MAKEFILE=ON \
+		-DUSE_SHARED_MBEDTLS_LIBRARY=ON \
+		-DENABLE_PROGRAMS=OFF \
+		-DENABLE_TESTING=OFF \
+		-DCMAKE_C_FLAGS="-Wno-stringop-overflow -Wno-error=array-bounds" \
```

Comment:
It is quite common that warnings pop in "distribution builds" as they use different compilers than the upstream CI, thus `-Werror` in non-CI builds is very frowned upon. 

I would prefer to not add a new section for `linux-aarch64` here but put the `-Wno-error=array-bounds` into the default case.